search-ui/src/requests/search-api.ts

```diff
@@ -2,16 +2,16 @@ import { axios } from '@/utils'
 
 // Interfaces
 import { SearchResponseI, SuggestionResponseI } from '@/interfaces'
-import { BusinessStatuses, BusinessTypes, SuggestionTypes } from '@/enums'
+import { BusinessStatuses, BusinessTypes } from '@/enums'
 
-//const HttpStatus = require('http-status-codes')
+const AUTO_SUGGEST_RESULT_SIZE = 20
 
 export async function getAutoComplete(searchValue: string): Promise<any> {
```

Comment:
can you change Promise<any> to Promise<SuggestionResponseI> so that code using this fn will translate the typing?